bindata/network/openshift-sdn/sdn-ovs.yaml

```diff
@@ -86,18 +86,25 @@ spec:
           fi
 
           # And finally start the ovs-vswitchd now the DB is prepped
-          /usr/share/openvswitch/scripts/ovs-ctl start --ovs-user=openvswitch:openvswitch --no-ovsdb-server --system-id=random
+          /usr/share/openvswitch/scripts/ovs-ctl start --ovs-user=openvswitch:openvswitch --no-ovsdb-server --system-id=random --no-monitor
 
+          /usr/share/openvswitch/scripts/ovs-ctl status > /dev/null &&
+          /usr/bin/ovs-appctl -T 5 ofproto/list > /dev/null &&
+          /usr/bin/ovs-vsctl -t 5 --retry show > /dev/null
+          
           # Load any flows that we saved
-          echo "info: Loading previous flows ..." 2>&1
+          echo "$TIMESTAMP info: Loading previous flows ..." 2>&1
           if [[ -f /var/run/openvswitch/flows.sh ]]; then
-             echo "info: Execute flow script ..." 2>&1
+             echo "$TIMESTAMP info: Adding br0 if it doesn't exist ..." 2>&1
+             /usr/bin/ovs-vsctl --may-exist add-br br0 -- set Bridge br0 fail_mode=secure protocols=OpenFlow13
```

Comment:
I don't like having the startup script have knowledge about what options the openshift-sdn binary sets on `br0`. This can be fixed later, but we ought to leave a bug/issue/card around reminding us about it.

Reply:
Created https://github.com/openshift/cluster-network-operator/issues/555. 
I understand the issue from maintainability perspective. Is there another reason why you don't like this?

Reply:
Just maintainability; if we change the details in openshift-sdn now, we'd have to also remember to change them in sync here

Reply:
@abhat @danwinship should we add a comment here that about that?